core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/application/ApplicationDesc.java

```diff
@@ -31,6 +32,8 @@
  * @author <a href="mailto:jmaxwell@users.sourceforge.net">Jon A. Maxwell (JAM)</a> - initial author
  */
 public class ApplicationDesc implements EntryPoint {
+    public static final String APPLICATION_DESC_ELEMENT = "application-desc";
+    public static final String JAVAFX_DESC_ELEMENT = "javafx-desc";
```

Comment:
Can we create an interface with all the *DESC* constants?

Reply:
I wouldn't do that right now and see no benefit. I would prefer keeping the element/attribute constants along with its class representation instead of having them isolated in one interface class. So each descriptor defines the namespace of its element/attributes. Usually there is also only one constant for the element name. As for JavaFX, however, I cannot find any "javafx-desc" element description in the JSR spec this is kind of a special case (as ApplicationDesc has a "type" and according to spec this that can be JavaFX)...